sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -184,7 +185,11 @@ def __init__(
       control_address,
       provision_info,
       worker_id=None):
-    self._worker_command_line = worker_command_line
+    if sys.platform == 'win32' and isinstance(worker_command_line, bytes):
+      # Fix subprocess.Popen not support bytes args
+      self._worker_command_line = worker_command_line.decode()
```

Comment:
is it possible that decoding will work both on windows and other systems? If so then maybe just always call decode.

Reply:
I believe so, inside the sdk the bytes are encoded here with utf-8:
https://github.com/apache/beam/blob/60581e8b1b6e93889cce78542e99d1fea4105d54/sdks/python/apache_beam/transforms/environments.py#L729
And decode utf-8 bytes should be safe as long as user not call it from elsewhere (e.g. a non utf-8 locale environment and run the subprocess worker manually, which should not be a normal use case).